src/runtime/handle.ts

```diff
@@ -70,6 +69,8 @@ export abstract class HandleOld {
   readonly options: HandleOptions;
   entityClass: EntityClass|null;
 
+  static noOpStorageAllocator = null;
```

Comment:
Could we add a storage allocator interface type without much work? It might let us avoid the implicit any here and the explicit any later.

Reply:
Yeah, probably. I didn't worry too much about it because I figured HandleOld will disappear once StorageNG is done. But I don't know, so I'll do it.

Reply:
Thanks, hopefully it will also make it easier to find when we remove HandleOld.

Reply:
Done.

Reply:
Thanks for nudging me to do the right thing.